src/utils.ts

```diff
@@ -3,6 +3,10 @@ import get = require('lodash/get');
 import isObject = require('lodash/isObject');
 import * as Color from 'color';
 
+interface ThemedDefaultProps extends DefaultProps {
```

Comment:
This could be `ThemedStyledProps<DefaultProps, Theme>` or `DefaultProps & ThemeProps<Theme>` in case we want to use existing interfaces.